inst/extdata/arg_data/README.md

```diff
@@ -0,0 +1,15 @@
+18.4.2024 Chouaib & Leo
+
+This folder includes data downloaded from the following paper
+https://www.nature.com/articles/s41467-023-36633-7
+
+And these URL sources:
+ 
+1) Sample metadata
+https://github.com/kihyunee/gut_resistotype/blob/data/sfile-microbiome-sample-detail.tsv
+
+2) read-based normalized ARG abundance table
+https://zenodo.org/records/7383076/files/cpg_table.20220516_scg_valid_only.tsv
+
+3) assembly-based normalized ARG abundance table
+https://zenodo.org/records/7383076/files/DS3.SCG_normalized_ARG_abund.columns_CARD_ref.n_6104.tsv
```

Comment:
There were two more files - could them be linked here, too?

Reply:
One I have downloaded from what seems a personal drive, the second also was downloaded without a source or link. Do you have the links?

Reply:
Ah, ok. List the file names here, and write "Received from Kihyun Lee on April 19, 2024 (personal communication)."